cli/src/main/resources/application.yml

```diff
@@ -38,6 +38,22 @@ message:
 
 ---
 
+spring:
+  profiles: receiver-kafka
+
+hono:
+  kafka:
+    consumerConfig:
+      bootstrap.servers: localhost:9092
+      group.id: hono-cli
+      auto.offset.reset: earliest
```

Comment:
We need to be careful about this. On the one hand I think the intention was not to miss any messages if the offset on a consumer somehow "got lost".  

On the other hand however this may lead to production problems, depending on how long we keep messages in topics: if we keep a million messages and the offset got lost when a consumer is restarted it will re-process everything from the very beginning. This can potentially take a very long time and seem to a client like a "downtime" as new messages are not yet processed.

Reply:
Ah just noticed it is for the CLI. Still I think using latest might be better ;-)

Reply:
> Ah just noticed it is for the CLI. Still I think using latest might be better ;-)

Yes, I do not see a productive use case for the CLI. I used `earliest` because it is easier for beginners to play around and immediately see messages produced before the consumer started. But you are right, it might be better to play it safe even in a beginner's tutorial.